### DISCUSSION TITLE
C'est quoi Http Client ? | Bonjour Angular

### DISCUSSION THREAD

**Discussion by giscus** [bonjour-angular/apprendre#29] [General]
# C'est quoi Http Client ? | Bonjour Angular

C'est quoi Http Client dans Angular ?

https://apprendre.bonjour-angular.com/cest-quoi/http-client/

**Comment by fabienmacip** [comment]
Merci pour ces rappels.
Je ne sais plus ce que permet le "takeUntilDestroyed" dans le pipe de la méthode "delete".
Help ? :-)
